gammapy/data/observations.py

```diff
@@ -359,11 +359,22 @@ def check(self, checks="all"):
         return checker.run(checks=checks)
 
 
-class ObservationList(UserList):
-    """List of `~gammapy.data.DataStoreObservation`.
+class Observations(object):
+    """Container class that holds a list of observations.
 
-    Could be extended to hold a more generic class of observations.
+    Parameters:
+    ----------
+    obs_list : list
+        A list of `~gammapy.data.DataStoreObservation`
     """
+    def __init__(self, obs_list=None):
+        self.obs_list = obs_list or []
```

Comment:
Maybe change to `self._obs_list` to have the as a private implementation detail?
That will force us to offer a good API on the `Observations` object, instead of accessing and manipulating `observations.obs_list` from all over the place.

There should be a `UserList` import at the top which can now be removed.

Reply:
> Maybe change to `self._obs_list` to have the as a private implementation detail?
+1 done
> There should be a `UserList` import at the top which can now be removed.
Done, i did some cleanup in the imports as well